servers/rendering/renderer_rd/shaders/ffx_a.h

```diff
@@ -0,0 +1,4987 @@
+//==============================================================================================================================
```

Comment:
Note that it's not a correct beginning of the (non-third party) header file which makes static check failed. You can take a proper header from any other header file across the engine.